app/helpers/lookup-action-type.js

```diff
@@ -31,7 +31,7 @@ export const actions = [
   ['HS', 'Special District/Mall Plan/REMIC NPA', ''],
   ['HU', 'Urban Renewal Plan and Amendments', ''],
   ['HZ', 'Preliminary Site Approval Application', ''],
-  ['LD', 'Legal Document (NOC, NOR, RD)', 'Notice of Certification (NOC), Notice of Restriction (NOR), Restrictive Declaration (RD)'],
+  ['LD', 'Legal Document (NOC, NOR, RD)', 'An LD action indicates that a project has one or more additional actions related to associated legal documents (such as a "Restrictive Declaration" RD, "Notice of Restriction" NOR, or "Notices of Certification" NOC) that impose additional property controls beyond zoning regulations. Such additional actions could be, for example, that a legal document is being created or modified, or that there is a follow-up approval pursuant to an existing legal document being processed.'],
```

Comment:
Does the tooltip render okay with the double quotes in this block of text?

Reply:
<img width="349" alt="Screen Shot 2020-02-20 at 4 47 53 PM" src="https://user-images.githubusercontent.com/26672885/74981718-c4666700-5400-11ea-8836-18de2f5f428b.png">

yep!